backend/infrahub/core/diff/model/path.py

```diff
@@ -188,6 +194,12 @@ class EnrichedDiffNode(BaseSummary):
     def __hash__(self) -> int:
         return hash(self.uuid)
 
+    def get_parent_node(self) -> EnrichedDiffNode | None:
+        for r in self.relationships:
+            for n in r.nodes:
+                return n
+        return None
```

Comment:
maybe `EnrichedDiffRelationship.nodes` should become `parent_node: EnrichedDiffNode | None` now?
but we can also leave this as is